File: 0520. Detect Capital/Solution0520.java
class Solution0520 {
    public boolean detectCapitalUse(String word) {
        int cap = 0;
        boolean first = false;
        for (int i = 0; i < word.length(); i++) {
            if (i == 0 && Character.isUpperCase(word.charAt(i))) {
                first = true;
            }
            if (Character.isUpperCase(word.charAt(i))) {
                cap++;
            }
        }
        return (cap == 0 || cap == word.length() || (cap == 1 && first));
    }
}

File: 0520. Detect Capital/Solution0520.js
/**
 * @param {string} word
 * @return {boolean}
 */
var detectCapitalUse = function(word) {
    let cap = 0;
    let first = false;
    for (let i = 0; i < word.length; i++) {
        if (i == 0 && word[i].toUpperCase()==word[i]) {
            first = true;
        }
        if (word[i].toUpperCase()==word[i]) {
            cap++;
        }
    }
    return (cap == 0 || cap == word.length || (cap == 1 && first));      
};